packages/gitmoji-changelog-cli/src/cli.js

```diff
@@ -29,6 +29,12 @@ async function main(options = {}) {
     }
   } catch (e) { /* ignore error */ }
 
+  if (options.groupSimilarCommits) {
+    logger.warn('⚗️  You are using a beta option, please be patient if it doesn\'t works as expected')
```

Comment:
work ?

Reply:
Or maybe:
```
⚗️  GroupSimilarCommits is an experimental feature - may not working as expected'
```

Reply:
I wanted to have a common message for all future beta options.
So we could have: `if (BETA_OPTIONS.includes(options))` -> text

Reply:
`You are using a beta feature - may not working as expected` ?

Reply:
Yes seems good to me